controllers/chat/getSignedChannelList.js

```diff
@@ -40,7 +60,7 @@ const getSignedChannelList = async (req, res) => {
   const queriedChannels = [];
   const promisedChannels = [];
 
-  const client = GetstreamSingleton.getChatInstance();
+  const client = new StreamChat(process.env.API_KEY, process.env.API_SECRET);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change replaces the singleton instance of `StreamChat` with a new instance for each request. This could potentially lead to performance issues if there are many concurrent requests, as each will create a new connection to the StreamChat service. If the singleton was used to limit the number of connections, this change might not be advisable. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->